src/UnitOfWork.php

```diff
@@ -888,7 +868,8 @@ private function updateNodes(): void
         $statements = [];
         foreach ($this->nodesScheduledForUpdate as $entity) {
             $this->traverseRelationshipEntities($entity);
-            $statements[] = $this->getPersister($entity::class)->getUpdateQuery($entity);
+            $statements[] = $this->entityManager->getEntityPersister($entity::class)->getUpdateQuery($entity);
+            $statements[] = $this->entityManager->getEntityPersister($entity::class)->getUpdateQuery($entity);
```

Comment:
Can you please clarify purpose of this duplicate?

Reply:
fixed